src/Microsoft.Identity.Client/AppConfig/AadAuthorityAudience.cs

```diff
@@ -58,22 +60,25 @@ public enum AadAuthorityAudience
         Default,
```

Comment:
It's not a common pattern to have a Default member in enums. Methods accepting this enum as param can enforce a value.

Reply:
I second Bogdan on this one. I'm not a big fan of these None

Reply:
"None" is a standard enum value. I find it difficult to understand what Default does.

Reply:
I'm good removing Default

---
In reply to: [248320157](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/790#discussion_r248320157) [](ancestors = 248320157)